src/changes-backwards-compatible.apib

```diff
@@ -10,6 +10,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 - We added `quotations.create`.
 
+- We added `quotations.update`.
```

Comment:
Can we summarise these three changes into one? They're all about adding quotations endpoints.